OrbitCore/FunctionFramepointerValidatorTest.cpp

```diff
@@ -11,7 +11,7 @@ namespace {
 
 constexpr uint8_t kFunctionWithFP[] = {0x55, 0x89, 0xE5, 0x83, 0xC0,
                                        0x01, 0xE8, 0x77, 0x00, 0x00,
-                                       0x00, 0x89, 0xEC, 0x5D};
+                                       0x00, 0x89, 0xEC, 0x5D, 0xC3};
 constexpr uint8_t kFunctionWithoutFP[] = {0x29, 0x25, 0x00, 0x00, 0x00, 0x00,
                                           0xE8, 0xFD, 0xFF, 0xFF, 0xFF, 0x01,
                                           0x25, 0x00, 0x00, 0x00, 0x00, 0xC3};
```

Comment:
do you want to add test for ENTER and LEAVE - including invalid ones maybe, depending on what we decide.

Reply:
Added a test for this and also for tail calls. Thanks.